ansible/group_vars/all/piecewise_global_config.yml

```diff
@@ -15,7 +15,7 @@ project_name: piecewise
 
 # Information about the fork and branch to deploy.
 # Specify any valid git remote (typically a github username)
-remote: https://github.com/opentechinstitute
+remote: https://github.com/critzo
```

Comment:
Did you mean for this, and a good number of the changes below, to get in the PR?